connectors/aws/src/test/scala/com/raphtory/aws/AwsSpoutTest.scala

```diff
@@ -23,7 +26,7 @@ object AwsSpoutTest extends IOApp {
 
     val spout  = AwsS3Spout(awsS3SpoutBucketName, awsS3SpoutBucketKey)
     val output = AwsS3Sink(awsS3OutputFormatBucketName)
-    val source = Source(spout, LotrGraphBuilder)
+    val source = CSVEdgeListSource(spout)
```

Comment:
Nice! It will be great to test this with some of the other spouts as well as we create them.